examples/basic-dynamic/analyzeDynamically.ts

```diff
@@ -0,0 +1,216 @@
+import {
+  PositionResult,
+  AnalysisResults,
+  LumpedMassAssignment,
+  PropertyAssignment,
+} from "../../awatif-data-structure";
+import { Node, Element, Assignment, Parameters } from "../../awatif-ui/";
+import { getTransformationMatrix } from "../../awatif-ui/src/utils/getTransformationMatrix.ts";
+import * as math from "mathjs";
+
+export function analyzeDynamically(
```

Comment:
Did you validate the correctness of the algorithm with the Python code you showed me earlier? the animation should match the double-pendulum result

Reply:
https://github.com/madil4/awatif/assets/38864087/735df4a0-5e0b-4acf-b90b-4772bcc75d62

It does based on multiple comparisons. We can do a more thorough validation if needed.